src/utils/ssi.ts

```diff
@@ -67,6 +68,63 @@ const canonizeCredential = async (
   return normalize(payload, options);
 };
 
+const canonizeProofOptions = async (
+  payload: Record<string, unknown>
+): Promise<string> => {
+  if (!isProof(payload))
+    throw new BadRequestError(indication.VERIFICATION_FAIL, {
+      detail: ApiErrorMessages.CANONIZE_BAD_PARAMS,
+    });
+  const options: Options.Normalize = {
+    algorithm: "URDNA2015",
+    format: "application/n-quads",
+    skipExpansion: false,
+  };
+  const context = {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://www.w3.org/2018/credentials/examples/v1",
+      "https://validatedid.github.io/jsonld-contexts/ades-signatures/v1/",
+    ],
+  };
+  // Delete signature or proof
+  (({ cades, ...o }) => o)(payload);
```

Comment:
You can do these 3 operations in a single line: `(({ cades, jws, proofValue, ...o }) => o)(payload)
`